src/test/components/ListOfPublishedProfiles.test.js

```diff
@@ -367,7 +371,7 @@ describe('ListOfPublishedProfiles', () => {
 
       // Clicking elsewhere should make the successful message disappear.
       fireFullClick((window: any));
-      expect(queryByText(/successfully/i)).toBe(null);
+      waitForElementToBeRemoved(queryByText(/successfully/i));
```

Comment:
In the previous code, I believe the text was changed (because the "state" changed) to an invalid text, even though the panel was still visible for a short period of time (because the list refresh is asynchronous). With the new code, the text doesn't change until the element is removed from the DOM.